upstash_vector/types.py

```diff
@@ -13,6 +13,11 @@ class Vector:
     vector: Union[List[float], SupportsToList]
     metadata: Optional[Dict] = None
 
+@dataclass
+class DataPayload:
```

Comment:
IMO, to be consistent with the name of `Vector` for the other type of payload, we should call this just `Data`